homeworks/nemethmate/1_introduction/2_vars_datatypes/my_caracter

```diff
@@ -0,0 +1,22 @@
+# képzeletbeli karakterem
+
+# inputok bekérése
+name = input("Hogy hívnak?").strip().capitalize() # név bekérése nagybetűvel és szóközök nélkül
+age = int(input("Hány éves vagy?").strip()) # kor bekérése és konvertálása egész számmá
+python_experiences = int(input("Hány éve foglalkozol phyton programozással?")) #Tapasztalak években
```

Comment:
Célszerűbb egy olyan változónak ami egyetlen egy értéket képvisel, egyes számú változó nevet adni. pl: python_experience, vagy python_exp_in_years, ez utóbbi még leíróbb.

Reply:
teljesen jogos. utóbbit hasznbálom is!